Source/WebKit/UIProcess/API/Cocoa/WKNavigationDelegatePrivate.h

```diff
@@ -104,6 +104,7 @@ static const WKNavigationResponsePolicy _WKNavigationResponsePolicyBecomeDownloa
 - (void)_webView:(WKWebView *)webView didFinishLoadWithRequest:(NSURLRequest *)request inFrame:(WKFrameInfo *)frame WK_API_AVAILABLE(macos(11.0), ios(14.0));
 
 - (void)_webView:(WKWebView *)webView didFailLoadDueToNetworkConnectionIntegrityWithURL:(NSURL *)url WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
+- (void)_webView:(WKWebView *)webView didChangeLookalikeCharactersFromURL:(NSURL *)sourceURL toURL:(NSURL *)destinationURL WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
Nit - same here, I would probably call this `Original`/`Adjusted` or `Old`/`New` instead of source and destination.

Reply:
I can make this change, but the reason I made it like this was to match `didPerformClientRedirectFromURL:(NSURL *)sourceURL toURL:(NSURL *)destinationURL` defined earlier in this file. Albeit the use for this is different, since it isn't actually redirect in our case.

Reply:
Ah, I see. I think it actually may be a tiny bit clearer to avoid using source/destination terminology here to avoid making it seem like the `sourceURL` is the currently loaded URL that we're navigating away from.